### PR TITLE
HBASE-25622 Result#compareResults should compare tags.

### PR DESCRIPTION
**Pull request by shahrs87**
https://issues.apache.org/jira/browse/HBASE-25622

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 13s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  1s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 23s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 43s |  master passed  |
| +1 :green_heart: |  compile  |   4m 59s |  master passed  |
| +1 :green_heart: |  checkstyle  |   1m 57s |  master passed  |
| +1 :green_heart: |  spotbugs  |   3m 40s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 14s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 26s |  the patch passed  |
| +1 :green_heart: |  compile  |   4m 53s |  the patch passed  |
| +1 :green_heart: |  javac  |   4m 53s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   1m 55s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  hadoopcheck  |  17m 31s |  Patch does not cause any errors with Hadoop 3.1.2 3.2.1 3.3.0.  |
| +1 :green_heart: |  spotbugs  |   4m  6s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 38s |  The patch does not generate ASF License warnings.  |
|  |   |  56m 29s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/3026 |
| Optional Tests | dupname asflicense javac spotbugs hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 3a9fe5581131 4.15.0-65-generic #74-Ubuntu SMP Tue Sep 17 17:06:04 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 830d2895b2 |
| Default Java | AdoptOpenJDK-1.8.0_282-b08 |
| Max. process+thread count | 96 (vs. ulimit of 30000) |
| modules | C: hbase-common hbase-client hbase-server U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/1/console |
| versions | git=2.17.1 maven=3.6.3 spotbugs=3.1.12 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 16s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  2s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 21s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m 46s |  master passed  |
| +1 :green_heart: |  compile  |   2m 10s |  master passed  |
| +1 :green_heart: |  shadedjars  |   7m 34s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 32s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 14s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   4m 39s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 14s |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 14s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   7m 25s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 31s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   1m 52s |  hbase-common in the patch passed.  |
| +1 :green_heart: |  unit  |   1m 17s |  hbase-client in the patch passed.  |
| +1 :green_heart: |  unit  | 197m 32s |  hbase-server in the patch passed.  |
|  |   | 236m 25s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/1/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/3026 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux a31635110b1c 4.15.0-128-generic #131-Ubuntu SMP Wed Dec 9 06:57:35 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 830d2895b2 |
| Default Java | AdoptOpenJDK-11.0.10+9 |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/1/testReport/ |
| Max. process+thread count | 3342 (vs. ulimit of 30000) |
| modules | C: hbase-common hbase-client hbase-server U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/1/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 10s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 21s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m  4s |  master passed  |
| +1 :green_heart: |  compile  |   1m 44s |  master passed  |
| +1 :green_heart: |  shadedjars  |   7m 12s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 18s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 13s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 53s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 48s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 48s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   7m 17s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 18s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   1m 53s |  hbase-common in the patch passed.  |
| +1 :green_heart: |  unit  |   1m 12s |  hbase-client in the patch passed.  |
| +1 :green_heart: |  unit  | 208m 55s |  hbase-server in the patch passed.  |
|  |   | 244m 22s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/1/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/3026 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 2e509e2fec77 4.15.0-136-generic #140-Ubuntu SMP Thu Jan 28 05:20:47 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 830d2895b2 |
| Default Java | AdoptOpenJDK-1.8.0_282-b08 |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/1/testReport/ |
| Max. process+thread count | 3691 (vs. ulimit of 30000) |
| modules | C: hbase-common hbase-client hbase-server U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/1/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by shahrs87** [comment]
@anoopsjohn  Here is the PR for master branch which applies cleanly to branch-2 also.
Could you please review ? Thank you !

**Comment by shahrs87** [comment]
@anoopsjohn  @virajjasani  Could you please help review this PR ? Thank you !

File: hbase-common/src/main/java/org/apache/hadoop/hbase/CellUtil.java [hunk 1]
```diff
@@ -570,6 +570,35 @@ public static boolean matchingValue(final Cell left, final byte[] buf) {
         buf.length);
   }
 
+  public static boolean matchingTags(final Cell left, final Cell right) {
+    return matchingTags(left, right, left.getTagsLength(), right.getTagsLength());
+  }
+
+  public static boolean matchingTags(final Cell left, final Cell right, int llength,
```
Comment (anoopsjohn) [inline hbase-common/src/main/java/org/apache/hadoop/hbase/CellUtil.java:577]: Can u pls keep this in PrivateCellUtil?

Comment (shahrs87) [reply hbase-common/src/main/java/org/apache/hadoop/hbase/CellUtil.java]: Done !

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 35s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 22s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 43s |  master passed  |
| +1 :green_heart: |  compile  |   4m 56s |  master passed  |
| +1 :green_heart: |  checkstyle  |   1m 58s |  master passed  |
| -1 :x: |  spotbugs  |   2m  3s |  hbase-server in master has 1 extant spotbugs warnings.  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 13s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 31s |  the patch passed  |
| +1 :green_heart: |  compile  |   4m 54s |  the patch passed  |
| +1 :green_heart: |  javac  |   4m 54s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   1m 57s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  hadoopcheck  |  17m 34s |  Patch does not cause any errors with Hadoop 3.1.2 3.2.1 3.3.0.  |
| +1 :green_heart: |  spotbugs  |   4m 21s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 40s |  The patch does not generate ASF License warnings.  |
|  |   |  57m 45s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/2/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/3026 |
| Optional Tests | dupname asflicense javac spotbugs hadoopcheck hbaseanti checkstyle compile |
| uname | Linux cdb3309f1b71 4.15.0-65-generic #74-Ubuntu SMP Tue Sep 17 17:06:04 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 0e6c2c419c |
| Default Java | AdoptOpenJDK-1.8.0_282-b08 |
| spotbugs | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/2/artifact/yetus-general-check/output/branch-spotbugs-hbase-server-warnings.html |
| Max. process+thread count | 96 (vs. ulimit of 30000) |
| modules | C: hbase-common hbase-client hbase-server U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/2/console |
| versions | git=2.17.1 maven=3.6.3 spotbugs=4.2.2 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   2m 45s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 32s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   5m 33s |  master passed  |
| +1 :green_heart: |  compile  |   2m 41s |  master passed  |
| +1 :green_heart: |  shadedjars  |   8m 30s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 49s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 18s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   5m 25s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 46s |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 46s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   8m 40s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 28s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   1m 50s |  hbase-common in the patch passed.  |
| +1 :green_heart: |  unit  |   1m 17s |  hbase-client in the patch passed.  |
| +1 :green_heart: |  unit  | 198m 47s |  hbase-server in the patch passed.  |
|  |   | 244m 57s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/2/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/3026 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 4e8b81057b91 4.15.0-128-generic #131-Ubuntu SMP Wed Dec 9 06:57:35 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 0e6c2c419c |
| Default Java | AdoptOpenJDK-11.0.10+9 |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/2/testReport/ |
| Max. process+thread count | 3474 (vs. ulimit of 30000) |
| modules | C: hbase-common hbase-client hbase-server U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/2/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   6m 52s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  5s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 27s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m 26s |  master passed  |
| +1 :green_heart: |  compile  |   1m 47s |  master passed  |
| +1 :green_heart: |  shadedjars  |   7m 22s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 22s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 14s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 52s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 45s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 45s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   7m 14s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 15s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   1m 53s |  hbase-common in the patch passed.  |
| +1 :green_heart: |  unit  |   1m 13s |  hbase-client in the patch passed.  |
| +1 :green_heart: |  unit  | 225m  6s |  hbase-server in the patch passed.  |
|  |   | 267m 25s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/2/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/3026 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 6b38e64c3083 4.15.0-136-generic #140-Ubuntu SMP Thu Jan 28 05:20:47 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 0e6c2c419c |
| Default Java | AdoptOpenJDK-1.8.0_282-b08 |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/2/testReport/ |
| Max. process+thread count | 3399 (vs. ulimit of 30000) |
| modules | C: hbase-common hbase-client hbase-server U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/2/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   2m 49s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 23s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 44s |  master passed  |
| +1 :green_heart: |  compile  |   4m 59s |  master passed  |
| +1 :green_heart: |  checkstyle  |   2m  0s |  master passed  |
| -1 :x: |  spotbugs  |   2m  3s |  hbase-server in master has 1 extant spotbugs warnings.  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 13s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 27s |  the patch passed  |
| +1 :green_heart: |  compile  |   4m 55s |  the patch passed  |
| +1 :green_heart: |  javac  |   4m 55s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   2m  0s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  hadoopcheck  |  17m 37s |  Patch does not cause any errors with Hadoop 3.1.2 3.2.1 3.3.0.  |
| +1 :green_heart: |  spotbugs  |   4m 19s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 42s |  The patch does not generate ASF License warnings.  |
|  |   |  59m  7s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/3/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/3026 |
| Optional Tests | dupname asflicense javac spotbugs hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 5a4be4c664a3 4.15.0-65-generic #74-Ubuntu SMP Tue Sep 17 17:06:04 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 0cc1ae48ed |
| Default Java | AdoptOpenJDK-1.8.0_282-b08 |
| spotbugs | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/3/artifact/yetus-general-check/output/branch-spotbugs-hbase-server-warnings.html |
| Max. process+thread count | 96 (vs. ulimit of 30000) |
| modules | C: hbase-common hbase-client hbase-server U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/3/console |
| versions | git=2.17.1 maven=3.6.3 spotbugs=4.2.2 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by shahrs87** [comment]
Spotbugs warning is not due to my changes.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 22s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  5s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 12s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m 35s |  master passed  |
| +1 :green_heart: |  compile  |   2m  7s |  master passed  |
| +1 :green_heart: |  shadedjars  |   7m 24s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 29s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 13s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   4m 38s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m  8s |  the patch passed  |
| +1 :green_heart: |  javac  |   2m  8s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   7m 25s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 29s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   1m 48s |  hbase-common in the patch passed.  |
| +1 :green_heart: |  unit  |   1m 19s |  hbase-client in the patch passed.  |
| +1 :green_heart: |  unit  | 198m 21s |  hbase-server in the patch passed.  |
|  |   | 236m 52s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/3/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/3026 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 9041509dc2b4 4.15.0-128-generic #131-Ubuntu SMP Wed Dec 9 06:57:35 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 0cc1ae48ed |
| Default Java | AdoptOpenJDK-11.0.10+9 |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/3/testReport/ |
| Max. process+thread count | 3341 (vs. ulimit of 30000) |
| modules | C: hbase-common hbase-client hbase-server U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/3/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 49s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 14s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m 40s |  master passed  |
| +1 :green_heart: |  compile  |   2m  6s |  master passed  |
| +1 :green_heart: |  shadedjars  |   8m  1s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 28s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 15s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   4m 36s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m  5s |  the patch passed  |
| +1 :green_heart: |  javac  |   2m  5s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   7m 47s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 26s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   2m  2s |  hbase-common in the patch passed.  |
| +1 :green_heart: |  unit  |   1m 22s |  hbase-client in the patch passed.  |
| +1 :green_heart: |  unit  | 225m 45s |  hbase-server in the patch passed.  |
|  |   | 266m 14s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/3/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/3026 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 4bfd6b63cf79 4.15.0-136-generic #140-Ubuntu SMP Thu Jan 28 05:20:47 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / 0cc1ae48ed |
| Default Java | AdoptOpenJDK-1.8.0_282-b08 |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/3/testReport/ |
| Max. process+thread count | 3237 (vs. ulimit of 30000) |
| modules | C: hbase-common hbase-client hbase-server U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3026/3/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Review by saintstack — APPROVED** [review]
This method is used verifying replication only -- and in tests. This extra compare is not being used in any critical path.

**Pull request merged**